lib/enhancements/edition.rb

```diff
@@ -43,6 +43,8 @@ def self.state_names
     )
   }
 
+  scope :published, -> { where(state: "published").desc(:created_at) }
```

Comment:
It looks like there's already a `published` scope in Edition. I wouldn't recommend redefining this with an unexpected ordering applied.

https://github.com/alphagov/govuk_content_models/blob/master/app/models/edition.rb#L38-L40

It'd probably be better to explicitly order in the report generator or add a scope with a different name.